src/panels/GeoLocationPanel/AnchorPanels/EarthPanel/types.ts

```diff
@@ -0,0 +1,28 @@
+// Incomplete type objects for the ArcGIS object we request for GeoLocationPanel
+// This was to avoid having to import the entire ArcGIS core since the @types file is
+// deprecated on npm.
```

Comment:
Add `anden` at hte end of this comment?  (Or, the id of whoever wrote it, so one knows who to ask about it?)